recipe/meta.yaml

```diff
@@ -1,27 +1,29 @@
 {% set name = "pytest-randomly" %}
-{% set version = "3.12.0" %}
+{% set version = "3.15.0" %}
+{% set name_url = name.split('-')[0]+"_"+name.split('-')[1] %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d60c2db71ac319aee0fc6c4110a7597d611a8b94a5590918bfa8583f00caccb2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name_url }}/{{ name_url }}-{{ version }}.tar.gz
+  sha256: b908529648667ba5e54723088edd6f82252f540cc340d748d1fa985539687047
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
```

Comment:
We should add `skip: true. # [py<38]` since that's what they officially support.